test/testcafe/framework/page-objects/EnrollOktaVerifyPageObject.js

```diff
@@ -3,7 +3,7 @@ import { userVariables, Selector } from 'testcafe';
 import { within } from '@testing-library/testcafe';
 
 const FORM_INFOBOX_ERROR_TITLE = '[data-se="o-form-error-container"] [data-se="callout"] > h3';
-const FORM_INFOBOX_ERROR_TITLE_V3 = '[data-se="callout"] > div > h2';
+const FORM_INFOBOX_ERROR_TITLE_V3 = '[data-se="callout"] div > h2';
```

Comment:
nit: I think we can assume that Odyssey Callout will only ever render one `h2` element so we can probably remove the `div >`